.github/workflows/push_pull.yml

```diff
@@ -38,6 +39,7 @@ jobs:
         with:
           asan: true
           ubsan: true
+          check_skip_long: true
```

Comment:
shouldn't it be the other way around? we don't have any time constraints for the nightly build, right?

Reply:
The Wang-Landau test can take an arbitrary amount of time depending on the load of the runner, and it can take so much resources that other tests can end up timing out too. We disabled it in our own GitLab sanitizer build, including in the nightly builds.

Reply:
but shouldn't we fix the issue instead of the symptom?

Reply:
We already did our best to reduce the runtime of the Wang-Landau test without sacrificing the statistical power of the check. The same is true for all statistical tests that are skipped by this command. The sanitizers are only needed on regular tests.